examples/screenshot-on-failure/tests/test.spec.ts

```diff
@@ -14,14 +14,10 @@
  * limitations under the License.
  */
 
-import { it, expect } from '@playwright/test';
+import { test, expect } from '@playwright/test';
 
-// Run this test with the '--param screenshotOnFailure' command line parameter
-// or 'npm run test'.
-
-it('is a basic test with the page', async ({ page, browserName, testInfo }) => {
+test('is a basic test with the page', async ({ page, browserName }) => {
   await page.setContent(`<div style="height: 500px; background-color: red">
```

Comment:
```suggestion
test('is a basic test with the page', async ({ page, browserName }, testInfo) => {
  console.log(testInfo.title)
```
why did you remove this? You can get it like in the suggestion.

Reply:
Because I can't find the `testInfo` in the second argument.

Reply:
Sorry, I misunderstood. I'll fix it soon.